libraries/plugins/xfpga/usrclk/fpga_user_clk.c

```diff
@@ -151,10 +153,29 @@ struct dfh {
 			uint64_t type : 4;
 		};
 	};
+	char guid_l[64];
```

Comment:
struct dfhv0 and  dfhv1 sturct used by other function in xfpga , so define dfhv0 and dfhv1 in types_int.h